kuma/wiki/templates/wiki/email/edited.ltxt

```diff
@@ -32,4 +32,8 @@ Edit Article:
 Article History:
 {% endtrans %}
  {{ history_url|absolutify }}
-{% if watch %}{{ unsubscribe_text(watch) }}{% endif %}{% endautoescape %}
+--
+{% if watch %}
+{% trans %}You are subscribed to edits on:{% endtrans %} {{ watch.content_object.title }}{% if watch.event_type == 'wiki edit document in tree' %}{% trans %} and all its sub-articles{% endtrans %}{% endif %}.
```

Comment:
A space is needed between the title and the phrase "all its sub-articles."  It may also make sense to display the URL instead of or in addition to the the title when the email is due to a tree subscription.

  From notification email:

```
--
You are subscribed to edits on: Web technology for developersand all its sub-articles.
Unsubscribe from these emails:
https://developer.mozilla.org/unsubscribe/9470?s=mkrtZZGAnr
```